lib/quiet_quality/logging.rb

```diff
@@ -0,0 +1,21 @@
+module QuietQuality
+  class Logging
```

Comment:
It's a pretty simple class, but I'd prefer to err on the side of too many specs - would you mind adding a unit-test file for this? testing it through the Options specs won't be as thorough, and I expect it to grow in those other PRs.